src/contexts/AuthContext.tsx

```diff
@@ -73,6 +79,24 @@ export const AuthProvider: React.FC<IProps> = ({ children }: IProps) => {
     },
     [onAuthenticated]
   );
+  const onRequestPasswordReset = useCallback(async (params: IResetPasswordParams) => {
+    const fn = UserAPI.requestResetPassword;
```

Comment:
why like this? if you were copying from `onAuthenticate` then you need to understand what you are doing. the `fn =` was because there were 2 potential functions to call. in your case there's only 1 so this code does not make sense

Reply:
removed this logic